### DISCUSSION TITLE
Audio breaks every 5-7s

### DISCUSSION THREAD

**Discussion by Alv777** [timschneeb/GalaxyBudsClient#347] [General]
So i recently bought the galaxy buds pro and they work very fine without the app, but when i start the buds client it works fine until you play any sound and the audio breaks for about 5s and then sounds fine and again 5s breaks and sounds fine again.

Im using the HP BT300 (Yes it's a printer adapter lol) on my computer and im in win11.

I really will love to solve this problem because im starting to love the app but this problem is really sad and makes the app unable to use.

But i really think its because of the bluetooth adapter.

**Comment by timschneeb** [comment]
Yes, this is most likely the Bluetooth adapter.

This app opens a second connection over Bluetooth to exchange status information about the earbuds and send commands back and forth. It's likely that the adapter can't handle the load.

**Comment by Alv777** [comment]
I bought a 4.0 Bluetooth adapter from tp-link and im getting same issue.

Which in the microsoft app works fine.

**Comment by Alv777** [comment]
Ok nevermind with the bt adapter works fine, thank you!
